lib/parser.js

```diff
@@ -252,14 +257,26 @@ module.exports = {
       var h = parsed.params.headers[i].toLowerCase();
       parsed.params.headers[i] = h;
 
-      if (h !== 'request-line') {
+      if (h === 'request-line') {
+        if (!options.strict) {
+          /**
```

Comment:
Skip the double '**' in the comment since this isn't a doxygen/javadoc/etc-style description.